src/policy/ast.rs

```diff
@@ -37,7 +37,7 @@ use super::serialize;
 /// given a witness.
 ///
 /// Furthermore, the policy can be normalized and is amenable to semantic analysis.
-#[derive(Clone, PartialEq, Eq, PartialOrd, Ord)]
+#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, std::hash::Hash)]
```

Comment:
In f37a401dbd982830233d1c2f8a36b8cd5748c6a1:

You should just say `Hash` here. I had no idea this fully-qualified syntax was legal, and it's not necessary/idiomatic.

Reply:
My goal was to differentiate `std::hash::Hash` from `hashes::Hash`, but just `Hash` is probably more canonical.